motr/m0crate/tests/test_btree.yaml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
2021 !!!

Reply:
Thanks

Reply:
You have only changed the copyright notice in this yaml file please change in the rest of the modified 7 files in this PR.